functional_tests/steps/bundles.py

```diff
@@ -121,3 +126,166 @@ def the_user_can_see_the_bundle_details_with_creator(context: Context) -> None:
 def bundle_inspect_show(context: Context) -> None:
     expect(context.page.get_by_text("Created by")).to_be_visible()
     expect(context.page.get_by_text(context.bundle_creator.get_username())).not_to_be_visible()
+
+
+# Bundles UI Data Setup
+@given("there are {no_bundles} bundles")
+def multiple_bundles_create(context: Context, no_bundles: str) -> None:
+    context.bundles = []
+    if no_bundles.isdigit():
+        for __ in range(int(no_bundles)):
+            context.bundles.append(BundleFactory(approved=False, created_by=context.user_data["user"]))
+
+
+@given("there are {no_preview_teams} Preview teams")
+def multiple_preview_teams_create(context: Context, no_preview_teams: str) -> None:
+    context.teams = []
+    if no_preview_teams.isdigit():
+        for preview_team_index in range(int(no_preview_teams)):
+            context.teams.append(
+                Team.objects.create(
+                    identifier="preview-team-" + str(preview_team_index), name="Preview_Team_" + str(preview_team_index)
```

Comment:
f-strings will be better here for readability:
```suggestion
                    identifier=f"preview-team-{preview_team_index}", name=f"Preview_Team_{preview_team_index}"
```

We could also perhaps drop the underscores?

```suggestion
                    identifier=f"preview-team-{preview_team_index}", name=f"Preview Team {preview_team_index}"
```